README.md

```diff
@@ -58,6 +68,12 @@ Next follow these instructions.
     source venv/bin/activate
     pip install -r requirements.txt
     ```
+    For Windows:
+    ```
+    virtualenv venv
+    venv\Scripts\activate
```

Comment:
```suggestion
    For Git Bash Users:
    ```
    source ./venv/Scripts/activate
     ``` 
   For Windows Command Line Users:   
    ``` venv\Scripts\activate```
```

@jeeva2812 mention about both the git bash and command-line users.